src/components/button/index.ts

```diff
@@ -1,7 +1,7 @@
 import { Component, EventEmitter, Input, Output } from '@angular/core';
 
 @Component({
-  selector: 'rio-button',
+  selector: 'sg-rio-button',
```

Comment:
Which we do - the prefix is 'rio-'.  We don't need a second prefix. Looks like you need to configure the prefix to use in the config.